packages/neos-ui/src/index.js

```diff
@@ -67,11 +67,16 @@ function * application() {
     yield system.getNeos;
 
     //
+    // Load frontend configuration very early, as we want to make it available in manifests
+    //
+    const frontendConfiguration = yield system.getFrontendConfiguration;
+
+    //
     // Initialize extensions
     //
     manifests
         .map(manifest => manifest[Object.keys(manifest)[0]])
-        .forEach(({bootstrap}) => bootstrap(globalRegistry, store));
+        .forEach(({bootstrap}) => bootstrap(globalRegistry, {store, frontendConfiguration}));
```

Comment:
I'm generally okay with the second parameter being an object here. But I'd prefer to initialize the `frontendConfigurationRegistry` (see L118) before running the manifests, so the frontendConfiguration can be accessed that way - since this our mechanism of providing globals.

Reply:
@grebaldi I think that won't work, because the manifest.js is *creating* the frontendConfiguration registry; so that'd be somehow a "chicken-egg" problem. See? :-)

What we could do is the following:
- pass the frontendConfiguration to the registries as above in my change
- move the initialization logic (L118ff) into the manifest.js, right where the frontendConfiguration registry is created.

This way, other registries could rely on frontend configuration being available and initialized in their manifest.

Reply:
Agreed, this would be the easiest way for now. But I think this shows an issue with our initialization logic... We should address that after the release.